code/modules/transport/transport_module.dm

```diff
@@ -762,9 +762,9 @@
 	glass = TRUE
 
 /obj/machinery/door/poddoor/lift/Initialize(mapload)
-	. = ..()
 	if(!isnull(transport_linked_id)) //linter and stuff
 		elevator_mode = TRUE
+	. = ..()
```

Comment:
```suggestion
	return ..()
```

Reply:
doors do globals and stuff on init if elevator mode is true